bucket/zeplin.json

```diff
@@ -0,0 +1,39 @@
+{
+    "homepage": "https://zeplin.io/",
+    "description": "Connected space for product teams. Handoff designs and styleguides with accurate specs, assets, code snippets—automatically.",
+    "license": "MIT",
```

Comment:
Is this just a client for product or you need to have valid license to use the software?

![image](https://user-images.githubusercontent.com/13260377/98695204-aa93f800-2372-11eb-9fea-406a9c95925d.png)

Reply:
I use it via my client's company work environment so I can't say for certain, but it looks like Zeplin only requires an account to work; I logged in with my Google account and it let me in straight away without any payment information.

Reply:
Good enough to add to scoop extras?

Reply:
It is definitely not MIT and it is definitely not free. The limitation for 1 project and account requirement is too strict.